charts/tidb-operator/templates/admission/admission-webhook-rbac.yaml

```diff
@@ -0,0 +1,140 @@
+{{- if eq .Values.admissionWebhook.create true }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.admissionWebhook.serviceAccount }}
```

Comment:
Better to have an `admissionWebhook.rbac.create` variable, otherwise the `serviceAccount` variable does not make sense (user cannot override the serviceAccount, after all), refer to `controller-manager.rbac` for detail.

Reply:
updated.